core/src/main/java/org/apache/calcite/prepare/PlannerImpl.java

```diff
@@ -213,8 +214,10 @@ public SqlNode validate(SqlNode sqlNode) throws ValidationException {
     ensure(State.STATE_3_PARSED);
     final SqlConformance conformance = conformance();
     final CalciteCatalogReader catalogReader = createCatalogReader();
+    final SqlOperatorTable opTab =
+        ChainedSqlOperatorTable.of(operatorTable, catalogReader);
     this.validator =
-        new CalciteSqlValidator(operatorTable, catalogReader, typeFactory,
```

Comment:
Shall we also update the code in `PlannerImpl#expandView` ?

Reply:
I have updated the code in a new commit. Thanks.